<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_65) on Thu Aug 20 14:53:31 CDT 2015 -->
<TITLE>
VarScan
</TITLE>

<META NAME="date" CONTENT="2015-08-20">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="VarScan";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../net/sf/varscan/package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/VarScan.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../index-files/index-1.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../net/sf/varscan/Trio.html" title="class in net.sf.varscan"><B>PREV CLASS</B></A>&nbsp;
&nbsp;NEXT CLASS</FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../index.html?net/sf/varscan/VarScan.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="VarScan.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
net.sf.varscan</FONT>
<BR>
Class VarScan</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../../resources/inherit.gif" ALT="extended by "><B>net.sf.varscan.VarScan</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>VarScan</B><DT>extends java.lang.Object</DL>
</PRE>

<P>
A set of tools for variant detection in next-generation sequence data.
<P>

<P>
<DL>
<DT><B>Version:</B></DT>
  <DD>2.4</DD>
<DT><B>Author:</B></DT>
  <DD>Daniel C. Koboldt <dkoboldt@genome.wustl.edu>

 <BR>
 <pre>
 COMMANDS
 pileup2snp [pileup file] OPTIONS
                        Call SNPs from a pileup file that meet certain cutoffs
                        Input:  Pileup file and parameters
                        Output: SNPs file with read counts and p-value

 pileup2indel [pileup file] OPTIONS
                        Call indels from a pileup file that meet certain cutoffs
                        Input:  Pileup file and parameters
                        Output: Indels file with read counts and p-value

 pileup2cns [pileup file] OPTIONS
                        Call consensus genotypes (reference or variant) at sites with sufficient coverage
                        Input:  Pileup file and parameters
                        Output: Consensus file with genotypes, read counts and p-values

 mpileup2cns [pileup file] OPTIONS
                        Call consensus genotypes (reference or variant) across one or more samples
                        Input:  SAMtools mpileup file and parameters
                        Output: Consensus file with genotypes, read counts and p-values, or VCF file

 somatic [normal_pileup] [tumor_pileup] [output] OPTIONS
                        Determine somatic status of SNPs from normal/tumor pileup for positions
                        Input:  Normal pileup, tumor pileup, and positions file
                        Output: SNPs file with read counts and somatic status

 readcounts [pileup] --variants-file [positions] --output-file [output]
                        Obtain read counts for each allele of variants from a pileup file
                        Input:  Variants file and pileupfile
                        Output: Variants file with read counts for each allele

 filter [variant file] OPTIONS
                        Filter a set of SNPs/indels based on coverage, reads, p-value, etc.
                        Input:  SNPs file with read counts and p-value
                        Output: Filtered SNPs file with read counts and p-value

 somaticFilter [somatic-status file] OPTIONS
                        Filter VarScan Somatic/Germline/LOH calls for clusters and proximal indels
                        Input:  VarScan output for SNPs or Indels (varscan.output.snp)
                        Output: Variants passing all filters (varscan.output.snp.filter)

 fpFilter [variant-file] [readcount-file] OPTIONS
                        Apply the false-positive filter to VarScan variant calls
                        Input:  VarScan output for SNPs or Indels (varscan.output.snp) and bam-readcount output
                        Output: Variants passing all filters (varscan.output.snp.fpfilter)

 processSomatic [somatic-status file] OPTIONS
                        Process VarScan output by somatic status and confidence
                        Input:  VarScan output for SNPs or Indels (varscan.output.snp)
                        Output: Variants by somatic status (varscan.output.snp.Somatic)

 copyCaller [copynumber file] OPTIONS
                        Process VarScan copynumber output to adjust for GC and make preliminary calls
                        Input:  VarScan copynumber output (varscan.output.copynumber)
                        Output: Normalized copy number with preliminary calls (varscan.output.copynumber.called)

 compare [file1] [file2] [type] [output] OPTIONS
                        Compares chromosome-position entries in two tab-delimited files
                        Input:  File 1 and File 2
                        Output: Merged, intersected, or unique entries

 limit [variants] --regions-file [regions] --output-file [output]
                        Limit a tab-delimited file (SNPs, pileup, etc) to a set of positions or regions
                        Input:  tab-delimited input file with chromosome & position; positions-file or regions-file
                        Output: Entries in input-file matching regions or positions

 coverage [pileup-file] --regions-file [regions] --output-file [output]
                        **Experimental** Calculate Q>20 coverage depth/breadth for a set of target regions
                        Input:  Pileup file and tab-delimited regions-file
                        Output: Coverage report at various Q>20 depths (1x,10x,20x...)


 </pre></DD>
</DL>
<HR>

<P>

<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#VarScan()">VarScan</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#compare(java.lang.String[], java.util.HashMap)">compare</A></B>(java.lang.String[]&nbsp;args,
        java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Compares two lists of positions/variants</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#copyCaller(java.lang.String[], java.util.HashMap)">copyCaller</A></B>(java.lang.String[]&nbsp;args,
           java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calls somatic copy number events from copynumber output</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#copynumber(java.lang.String[], java.util.HashMap)">copynumber</A></B>(java.lang.String[]&nbsp;args,
           java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Determines tumor copy number from normal and tumor pileup files</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#coverage(java.lang.String[], java.util.HashMap)">coverage</A></B>(java.lang.String[]&nbsp;args,
         java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Reports region coverage from a BAM file</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#filter(java.lang.String[], java.util.HashMap)">filter</A></B>(java.lang.String[]&nbsp;args,
       java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Filters variants by coverage, significance, frequency, etc.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#fpfilter(java.lang.String[], java.util.HashMap)">fpfilter</A></B>(java.lang.String[]&nbsp;args,
         java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Applies false positive filter using bam-readcount information</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#getSignificance(int, int)">getSignificance</A></B>(int&nbsp;obsReads1,
                int&nbsp;obsReads2)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calculates significance of read counts versus baseline error</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#getSignificance(int, int, int, int)">getSignificance</A></B>(int&nbsp;expReads1,
                int&nbsp;expReads2,
                int&nbsp;obsReads1,
                int&nbsp;obsReads2)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calculates significance of read counts between two samples</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#limit(java.lang.String[], java.util.HashMap)">limit</A></B>(java.lang.String[]&nbsp;args,
      java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Limits pileup or variant files to a list of positions or regions</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#main(java.lang.String[])">main</A></B>(java.lang.String[]&nbsp;args)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Runs the main execution logic</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#mpileup2call(java.lang.String[], java.util.HashMap, java.lang.String)">mpileup2call</A></B>(java.lang.String[]&nbsp;args,
             java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
             java.lang.String&nbsp;callType)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calls SNPs from an mpileup file</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#pileup2call(java.lang.String[], java.util.HashMap, java.lang.String)">pileup2call</A></B>(java.lang.String[]&nbsp;args,
            java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
            java.lang.String&nbsp;callType)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calls SNPs from a pileup file</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#processSomatic(java.lang.String[], java.util.HashMap)">processSomatic</A></B>(java.lang.String[]&nbsp;args,
               java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Splits VarScan output according to somatic status and confidence</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#readcounts(java.lang.String[], java.util.HashMap)">readcounts</A></B>(java.lang.String[]&nbsp;args,
           java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Obtains read counts for a list of variants</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#somatic(java.lang.String[], java.util.HashMap)">somatic</A></B>(java.lang.String[]&nbsp;args,
        java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calls somatic/germline/LOH variants from normal and tumor pileup files</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#somaticFilter(java.lang.String[], java.util.HashMap)">somaticFilter</A></B>(java.lang.String[]&nbsp;args,
              java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Filters variants by coverage, significance, frequency, etc.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../net/sf/varscan/VarScan.html#trio(java.lang.String[], java.util.HashMap, java.lang.String)">trio</A></B>(java.lang.String[]&nbsp;args,
     java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
     java.lang.String&nbsp;callType)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calls SNPs in a father-mother-child trio from an mpileup file</TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>equals, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="VarScan()"><!-- --></A><H3>
VarScan</H3>
<PRE>
public <B>VarScan</B>()</PRE>
<DL>
</DL>

<!-- ============ METHOD DETAIL ========== -->

<A NAME="method_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Method Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="main(java.lang.String[])"><!-- --></A><H3>
main</H3>
<PRE>
public static void <B>main</B>(java.lang.String[]&nbsp;args)</PRE>
<DL>
<DD>Runs the main execution logic
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="pileup2call(java.lang.String[], java.util.HashMap, java.lang.String)"><!-- --></A><H3>
pileup2call</H3>
<PRE>
public static void <B>pileup2call</B>(java.lang.String[]&nbsp;args,
                               java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
                               java.lang.String&nbsp;callType)</PRE>
<DL>
<DD>Calls SNPs from a pileup file
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments and parameters<DD><CODE>callType</CODE> - "SNP", "INDEL", or "CNS"</DL>
</DD>
</DL>
<HR>

<A NAME="mpileup2call(java.lang.String[], java.util.HashMap, java.lang.String)"><!-- --></A><H3>
mpileup2call</H3>
<PRE>
public static void <B>mpileup2call</B>(java.lang.String[]&nbsp;args,
                                java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
                                java.lang.String&nbsp;callType)</PRE>
<DL>
<DD>Calls SNPs from an mpileup file
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments and parameters<DD><CODE>callType</CODE> - "SNP", "INDEL", or "CNS"</DL>
</DD>
</DL>
<HR>

<A NAME="readcounts(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
readcounts</H3>
<PRE>
public static void <B>readcounts</B>(java.lang.String[]&nbsp;args,
                              java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Obtains read counts for a list of variants
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="somatic(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
somatic</H3>
<PRE>
public static void <B>somatic</B>(java.lang.String[]&nbsp;args,
                           java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Calls somatic/germline/LOH variants from normal and tumor pileup files
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="trio(java.lang.String[], java.util.HashMap, java.lang.String)"><!-- --></A><H3>
trio</H3>
<PRE>
public static void <B>trio</B>(java.lang.String[]&nbsp;args,
                        java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params,
                        java.lang.String&nbsp;callType)</PRE>
<DL>
<DD>Calls SNPs in a father-mother-child trio from an mpileup file
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments and parameters<DD><CODE>callType</CODE> - "SNP", "INDEL", or "CNS"</DL>
</DD>
</DL>
<HR>

<A NAME="copynumber(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
copynumber</H3>
<PRE>
public static void <B>copynumber</B>(java.lang.String[]&nbsp;args,
                              java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Determines tumor copy number from normal and tumor pileup files
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="filter(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
filter</H3>
<PRE>
public static void <B>filter</B>(java.lang.String[]&nbsp;args,
                          java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Filters variants by coverage, significance, frequency, etc.
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="fpfilter(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
fpfilter</H3>
<PRE>
public static void <B>fpfilter</B>(java.lang.String[]&nbsp;args,
                            java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Applies false positive filter using bam-readcount information
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="somaticFilter(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
somaticFilter</H3>
<PRE>
public static void <B>somaticFilter</B>(java.lang.String[]&nbsp;args,
                                 java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Filters variants by coverage, significance, frequency, etc.
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="processSomatic(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
processSomatic</H3>
<PRE>
public static void <B>processSomatic</B>(java.lang.String[]&nbsp;args,
                                  java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Splits VarScan output according to somatic status and confidence
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="copyCaller(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
copyCaller</H3>
<PRE>
public static void <B>copyCaller</B>(java.lang.String[]&nbsp;args,
                              java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Calls somatic copy number events from copynumber output
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="compare(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
compare</H3>
<PRE>
public static void <B>compare</B>(java.lang.String[]&nbsp;args,
                           java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Compares two lists of positions/variants
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="limit(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
limit</H3>
<PRE>
public static void <B>limit</B>(java.lang.String[]&nbsp;args,
                         java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Limits pileup or variant files to a list of positions or regions
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="coverage(java.lang.String[], java.util.HashMap)"><!-- --></A><H3>
coverage</H3>
<PRE>
public static void <B>coverage</B>(java.lang.String[]&nbsp;args,
                            java.util.HashMap&lt;java.lang.String,java.lang.String&gt;&nbsp;params)</PRE>
<DL>
<DD>Reports region coverage from a BAM file
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>args</CODE> - Command-line arguments</DL>
</DD>
</DL>
<HR>

<A NAME="getSignificance(int, int)"><!-- --></A><H3>
getSignificance</H3>
<PRE>
public static double <B>getSignificance</B>(int&nbsp;obsReads1,
                                     int&nbsp;obsReads2)</PRE>
<DL>
<DD>Calculates significance of read counts versus baseline error
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>obsReads1</CODE> - Reads supporting allele 1<DD><CODE>obsReads2</CODE> - Reads supporting allele 2
<DT><B>Returns:</B><DD>p-value P-value from Fisher's Exact Test</DL>
</DD>
</DL>
<HR>

<A NAME="getSignificance(int, int, int, int)"><!-- --></A><H3>
getSignificance</H3>
<PRE>
public static double <B>getSignificance</B>(int&nbsp;expReads1,
                                     int&nbsp;expReads2,
                                     int&nbsp;obsReads1,
                                     int&nbsp;obsReads2)</PRE>
<DL>
<DD>Calculates significance of read counts between two samples
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>expReads1</CODE> - Reads supporting allele 1 (expected)<DD><CODE>expReads2</CODE> - Reads supporting allele 2 (expected)<DD><CODE>obsReads1</CODE> - Reads supporting allele 1 (observed)<DD><CODE>obsReads2</CODE> - Reads supporting allele 2 (observed)
<DT><B>Returns:</B><DD>p-value         P-value from Fisher's Exact Test</DL>
</DD>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../net/sf/varscan/package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/VarScan.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../index-files/index-1.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../net/sf/varscan/Trio.html" title="class in net.sf.varscan"><B>PREV CLASS</B></A>&nbsp;
&nbsp;NEXT CLASS</FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../index.html?net/sf/varscan/VarScan.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="VarScan.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
